ketcher-autotests/tests/Indigo-Tools/Aromatize-Dearomatize/aromatize-dearomatize.spec.ts

```diff
@@ -378,4 +379,105 @@ test.describe('Aromatize/Dearomatize Tool', () => {
 
     expect(cmlFile).toEqual(cmlFileExpected);
   });
+
+  test('User can auromatize molecules with query parameters (not custom query, but only ordinary).', async ({
+    page,
+  }) => {
+    /*
+    Test case: https://github.com/epam/ketcher/issues/3849 - Test case 1
+    Description: User can auromatize molecules with query parameters (not custom query, but only ordinary).
+    1. Clear canvas
+    2. Open as New Project: AllPossibleQueryFeaturesWithOutCustomQuery.ket
+    3. Press Aromatize button
+    4. Validate canvas
+    IMPORTANT: Result of execution is incorrect because of https://github.com/epam/ketcher/issues/3529 issue.
+    Screenshots should be updated after fix.
+     */
+    await openFileAndAddToCanvasAsNewProject(
```

Comment:
I just want to clarify here, is it necessary to use it in test 'open as a new project'?

Reply:
Yep, it put on the canvas with clear canvas before